cron_main.py

```diff
@@ -63,5 +62,6 @@
                                ('/tasks/admin/clear_old_subs', AdminSubsClear),
                                ('/tasks/admin/enqueue/clear_old_webhooks', AdminWebhooksClearEnqueue),
                                ('/tasks/admin/clear_old_webhooks', AdminWebhooksClear),
+                               ('/tasks/admin/rebuild_district_teams/([0-9]+)', AdminCreateDistrictTeamsDo),
```

Comment:
I'm not sure, @phil-lopreiato told me to put them there. Do you think they should be somewhere else?  If so, please let me know to where you like me to move them.